### PR TITLE
S3linkcontroller

### PR DESCRIPTION
**Pull request by tinazhouhui**
Helloo
- just implemented two very simle tests for s3link controller and model :)

### CODE REVIEW

File: tests/controllers/S3LinkController.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    const request = {
+      body: {
+        frames: ['image22.jpg', 'image33.jpg']
+      }
+    } as Request;
+    const response = {
+      json: fn((data) => {
+        // const fileNames = Object.keys(data);
+
+        expect(data).toHaveLength(3);
```
Comment (Fav8) [inline tests/controllers/S3LinkController.ts:21]: everything looks good, I don't understand why data.length sould be 3 if we are passing 2 pictures
data right now should be ['image22link','image33link'] right?

File: tests/controllers/S3LinkController.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import S3LinkController from '../../src/controllers/S3LinkController';
+
+describe('S3 Link Controller', () => {
+  const container = containerBuilder();
+  const s3LinkController = container.get('s3LinkController') as S3LinkController;
+
+  test('run action Default', () => {
+    const request = {
+      body: {
+        frames: ['image22.jpg', 'image33.jpg']
```
Comment (Fav8) [inline tests/controllers/S3LinkController.ts:14]: we should get the name of the video too here right?

File: tests/controllers/S3LinkController.ts (hunk 1 above)
Comment (tinazhouhui) [reply tests/controllers/S3LinkController.ts:21]: because this is the faces not the images :)

File: tests/controllers/S3LinkController.ts (hunk 2 above)
Comment (tinazhouhui) [reply tests/controllers/S3LinkController.ts:14]: yes we should

**Review by Fav8 — APPROVED** [review]
approved closing

**Pull request merged**